src/messaging/tests/TestReliableMessageProtocol.cpp

```diff
@@ -1215,7 +1192,6 @@ void CheckLostResponseWithPiggyback(nlTestSuite * inSuite, void * inContext)
     mockSender.IsOnMessageReceivedCalled   = false;
     mockSender.mReceivedPiggybackAck       = false;
 
-    // 1 tick is 64 ms, sleep 65 ms to trigger re-transmit from sender
```

Comment:
The comment about how we are sleeping only long enough to trigger re-transmit from sender (only!) is pretty important here and should stay.